### ISSUE TITLE
2018.3 issues with directories

### ISSUE THREAD

**Issue by Miles-Tone** [LibPdIntegration/LibPdIntegration#4]
Hi there, me and Karl who posted the other issue recently used this as part of the GlobalGameJam, thanks for putting this up.
We came into a couple of issues we believe based around directories in the newer Unity versions.
As Karl has said the directory location needed to be changed.
There are issues with when it gets built, making the above changes makes it work on Mac as a build, but it doesn’t build on windows.
I will try and have a look when I get my windows machine back, but I’m not sure how far i’d get.
Thanks,
Miles

**Comment by NiallMoody** [comment]
Hi! Sorry, I only just realised I didn't have notifications turned on for issues, and I'd completely missed the small handful of issues people have posted.

[Application.dataPath](https://github.com/LibPdIntegration/LibPdIntegration/issues/3) is tied up with this, so I've closed that issue to keep everything in the one place. I should have some time in the next couple of weeks to investigate properly and figure out a good cross-platform solution.
